### DISCUSSION TITLE
File Upload of same file gives [500 Internal Server Error]

### DISCUSSION THREAD

**Discussion by sachin207** [eclipse-theia/theia#10196] [Q&A]
I am using theia 1.17.2 and its FileUploadServcie. 
On the first upload of the file, it is successful.
![image](https://user-images.githubusercontent.com/4499572/135384557-058dcfd1-56c7-4e15-9d05-4b851dcdbd57.png)

On subsequent upload of the same file, it gives a 500 Internal Server Error.
![image](https://user-images.githubusercontent.com/4499572/135385236-3cfcaf0a-e36b-446b-8ed4-0a3576031be3.png)

Is there a way to inform users about duplicate upload or replace it with the new file?

**Comment by msujew** [comment]
Hi @sachin207, thank you for reporting this issue. I created an appropriate bug report #10197. Discussion on that bug will continue there.
